examples/vllm/config-circuitbreaker.yml

```diff
@@ -0,0 +1,27 @@
+circuit_breakers:
+  - name: request_timeout_breaker
+    metrics:
+      matches:
+        - error.error_type == 'TimeoutError'
+    triggers:
+      - type: consecutive # break when there are 10 consecutive errors
+        threshold: 10
```

Comment:
Nice! I like this definition for circuit breakers. Is there a value for request timeout that we can set as well that gets passed along to the clients?